js/ui/handler/scroll_zoom.js

```diff
@@ -97,6 +97,7 @@ ScrollZoom.prototype = {
     },
 
     _zoom: function (delta) {
+        if (delta === 0) return;
```

Comment:
Being more robust always matters! It might fix similar problems on other platforms.